crates/graphql/src/schema.rs

```diff
@@ -1,15 +1,24 @@
+use std::{collections::HashMap, hash::Hash};
+
+use async_trait::async_trait;
+use dataloader::{non_cached::Loader, BatchFn};
 use indexer_core::{
     db::{
         models,
-        tables::{metadata_creators, metadatas},
-        Pool,
+        tables::{metadata_creators, metadata_jsons, metadatas},
+        Pool, PooledConnection,
     },
     prelude::*,
 };
-use juniper::{EmptySubscription, FieldResult, GraphQLInputObject, GraphQLObject, RootNode};
+use juniper::{EmptyMutation, EmptySubscription, GraphQLObject, RootNode};
+
+#[derive(Debug, Clone, GraphQLObject)]
+struct NftDetail {
+    description: String,
+    image: String,
+}
 
-#[derive(GraphQLObject)]
-#[graphql(description = "A Solana NFT")]
+#[derive(Debug, Clone)]
```

Comment:
```suggestion
#[derive(Debug, Clone)]
#[allow(dead_code)]
```

to silence the compiler warnings